app/src/main/java/com/example/next_show/fragments/ShowDetailFragment.java

```diff
@@ -91,6 +99,19 @@ private void setUpView(View currView) {
         // put into imageview -> if not loaded yet, this will be blank but loads properly if clicked again
         Glide.with(getContext()).load(currShow.getImageUrl()).into(ivShowPoster);
 
+        // go through show genres and add chips
+        List<String> showGenres = currShow.getGenres();
+        int track = 1;
```

Comment:
nit: track doesn't seem like a great variable name.  Something related to genreCount or something of the like would likely be better -- or genreChipCount